### PR TITLE
Added Embeds and updated README

### PR DESCRIPTION
**Pull request by thrzl**
### README updates
- Added Intents overview
- Added dockerless instructions
### Python File Updates
- Added Intents
- Post welcome messages to a channel as well as send a DM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,45 @@
 # discord.py-welcome-bot
 
-A simple bot that can be used to welcome new users to a discord server, sending them a custom private message and adding them to a role.
+A simple bot that can be used to welcome new users to a discord server, send them a custom private message, send a welcome message to a specific channel, and adding them to a role.
```
Comment (jcreek) [inline README.md]: I clearly typed the original in a rush 🤦 `and adding them to a role.` should be `and give them a specific role`

File: README.md [hunk 2]
```diff
@@ -1,12 +1,45 @@
...
 
 A video explaining how to set up everything except adding the user to a role can be found [here](https://youtu.be/N0NP7BfUFxA).
 
-**Update November 2019** - Now set up as a docker container to ensure full compatibility as I will not be maintaining this. Specific versions of Python and Discord.py are required for this script to work.
-
-## How to install the welcome-bot as a docker container on a server (raspberrypi)
+**Update November 2019** - Now set up as a docker container to ensure full compatibility as I will not be maintaining this. Specific versions of Python and Discord.py are required for this script to work.\
+**Update November 2020** - Added basic install instructions for home computers and generic Linux distros, and how to enable Intents.
+## Intents
```
Comment (jcreek) [inline README.md:12]: Markdown headings should have a blank line above and below them. I'm conscious I didn't do this for this project, and I am deeply ashamed.

File: README.md [hunk 3]
```diff
@@ -1,12 +1,45 @@
...
-A simple bot that can be used to welcome new users to a discord server, sending them a custom private message and adding them to a role.
+A simple bot that can be used to welcome new users to a discord server, send them a custom private message, send a welcome message to a specific channel, and adding them to a role.
 
 A video explaining how to set up everything except adding the user to a role can be found [here](https://youtu.be/N0NP7BfUFxA).
 
-**Update November 2019** - Now set up as a docker container to ensure full compatibility as I will not be maintaining this. Specific versions of Python and Discord.py are required for this script to work.
-
-## How to install the welcome-bot as a docker container on a server (raspberrypi)
+**Update November 2019** - Now set up as a docker container to ensure full compatibility as I will not be maintaining this. Specific versions of Python and Discord.py are required for this script to work.\
+**Update November 2020** - Added basic install instructions for home computers and generic Linux distros, and how to enable Intents.
```
Comment (jcreek) [inline README.md]: Please feel free to put your own username alongside this update - it's great work and I'm sure will help the people who are still using this bot, you deserve the credit 😃

File: welcome-bot.py [hunk 4]
```diff
@@ -4,9 +4,10 @@
 # import all necessary commands and libraries
 import discord
 import asyncio
-
-client=discord.Client()
-
+intents = discord.Intents(members=True)
+client=discord.Client(intents=intents)
+welcomechannel = await client.fetch_channel(channel_id) #https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID
```
Comment (jcreek) [inline welcome-bot.py]: Comment missing space after #

File: welcome-bot.py [hunk 5]
```diff
@@ -28,20 +29,26 @@ async def on_ready():
...
+    role = discord.utils.get(member.server.roles, name="name-of-your-role") # Gets the member role as a `role` object
+    await client.add_roles(member, role) # Gives the role to the user
     print("Added role '" + role.name + "' to " + member.name)
 
+@client.event
+async def on_member_leave(member):
+    print("Recognised that a member called " + member.name + " left")
+    embed=discord.Embed(
+        title="😢 Goodbye "+member.name+"!",
+        description="Until we meet again old friend." #A description isn't necessary, you can delete this line if you don't want a description.
```
Comment (jcreek) [inline welcome-bot.py]: Comment missing space after #

File: welcome-bot.py [hunk 6]
```diff
@@ -28,20 +29,26 @@ async def on_ready():
...
+        color=discord.Color.red() #There are lots of colors, you can check them here: https://discordpy.readthedocs.io/en/latest/api.html?highlight=discord%20color#discord.Colour
```
Comment (jcreek) [inline welcome-bot.py]: Comment missing space after #

**Comment by jcreek** [comment]
@isigebengu-mikey I've put some very minor comments on the PR, it looks great otherwise and I'd be delighted to merge it in once those changes are made :)

File: README.md (hunk 3 above)
Comment (thrzl) [reply README.md]: Thanks jcreek!

**Comment by thrzl** [comment]
Alright, is this better?

**Review by jcreek — APPROVED** [review]
Looks great 😃

**Pull request merged**